draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1014,6 +1014,30 @@ Internet-Draft                 ACTN POI                   November 2020
    [Editor's note:] Add YANG models used for tunnel and service
    configuration.
 
+3.3. PCEP
+
+   [RFC8623] examines the applicability of a Path Computation Element
+   (PCE) [RFC5440] and PCE Communication Protocol (PCEP) to the ACTN
+   framework. It further describes how the PCE architecture is
+   applicable to ACTN.  It also lists the PCEP extensions that are
```

Comment:
just to improve readability: It further describes how the PCE architecture is applicable to ACTN and lists the PCEP extensions...

Reply:
**2020-11-09 TEAS**

Ok